src/app/[locale]/_utils/common.ts

```diff
@@ -1,8 +1,20 @@
 import { LoginInfo } from '../_model/LoginInfo';
 import { MagicLink } from '../_model/MagicLink';
+import { ROUTES } from './routes';
 import { storageLocaleOps } from './storage';
 import { SessionState } from '@/api/generated/webProfile/SessionState';
 
+export type LoginTypes =
+  | 'login_to_SessionExit'
+  | 'login_to_Profile'
+  | 'login_to_UnlockAccessL2'
+  | 'login_to_UnlockAccessL3';
+
+export const FLOW_SESSION_EXIT = 'login_to_SessionExit';
```

Comment:
you could create a constant 'Flow', which is an object, containing all the values